.gitignore

```diff
@@ -51,7 +51,6 @@ proto/spec/**/*.pb.go
 *.aux
 *.bbl
 *.blg
-*.log
```

Comment:
I don't think that we want to delete this

Reply:
We do, as it already exists in that file